ckanext/switzerland/i18n/de/LC_MESSAGES/ckanext-switzerland.po

```diff
@@ -673,3 +680,133 @@ msgstr "Wissenschaftliche Arbeit"
 #: ckanext/switzerland/helpers.py:41
 msgid "Best practice"
 msgstr "Best practice"
+
+#: ckanext/switzerland/templates/package/dataset_showcase_list.html:17
+msgid "Associate this application with this dataset"
+msgstr ""
+
+#: ckanext/switzerland/templates/package/dataset_showcase_list.html:17
+msgid "Add to application"
+msgstr ""
+
+#: ckanext/switzerland/templates/package/dataset_showcase_list.html:21
+msgid "Applications featuring {dataset_name}"
+msgstr ""
+
+#: ckanext/switzerland/templates/package/dataset_showcase_list.html:26
+msgid "There are no applications that feature this dataset"
+msgstr ""
+
+#: ckanext/switzerland/templates/showcase/snippets/search_box.html:25
+msgid "Search applications..."
+msgstr "Applikationen suchen..."
+
+#: ckanext/switzerland/templates/showcase/snippets/showcase_info.html:55
+msgid "Datasets in Application"
+msgstr "Datensätze in Applikationen"
+
+#: ckanext/switzerland/templates/showcase/snippets/showcase_info.html:65
+msgid "There are no Datasets in this Application"
+msgstr "Es gibt keine Datensätze in diese Applikation"
+
+#: ckanext/switzerland/templates/showcase/snippets/showcase_item_small.html:30
+msgid "This application has no description"
+msgstr "Diese Applikation hat keine Beschreibung"
+
+#: ckanext/switzerland/templates/showcase/snippets/showcase_item_small.html:46
+msgid "Remove dataset from this application"
+msgstr ""
```

Comment:
Why are these not translated? This and the following messages?

Reply:
Oops, I missed out this translation (although i translated the other string from the same template). Added it back now.